src/main/java/com/dakar/dakar/resolvers/MutationResolver.java

```diff
@@ -27,6 +27,24 @@ public Journey createJourney(JourneyInput journeyInput) {
         return journeyService.saveJourney(journeyMono).blockFirst();
     }
 
+    /**
+     * Update a Journey
+     *
+     * @param journey
+     * @return The newly updated Journey
+     */
+    public Journey updateJourney(JourneyInput journey) {
+        Journey journeyCreated = new Journey(journey.getId(), journey.getPrice(), journey.getDestination(), "");
+        Mono<Journey> journeyFound = journeyService.findById(journey.getId())
+                .map((Journey it) -> {
+                    it.setDestination(journey.getDestination());
+                    it.setOwner(journey.getOwner());
+                    it.setPrice(journey.getPrice());
+                    return it;
+                });
+        return journeyService.saveJourney(journeyFound).blockFirst();
```

Comment:
what if journey found is null ? 
no new journey should be saved.
can we keep using the same pipline ?